index.test-d.ts

```diff
@@ -0,0 +1,67 @@
+import * as minimist from 'minimist';
```

Comment:
I'm not familiar with TS much, what's this file for?

Reply:
You test code with AVA. This tests the TS types, to ensure they're correct.

See https://github.com/SamVerschueren/tsd-check

Reply:
👍